neqo-transport/src/recovery/mod.rs

```diff
@@ -1526,15 +1562,18 @@ mod tests {
             assert_eq!(path.amplification_limit(), SPARE);
         }
 
-        lr.on_packet_sent(SentPacket::new(
-            PacketType::Initial,
-            0,
-            IpTosEcn::default(),
-            now(),
-            true,
-            Vec::new(),
-            ON_SENT_SIZE,
-        ));
+        lr.on_packet_sent(
+            SentPacket::new(
+                PacketType::Initial,
+                0,
+                IpTosEcn::default(),
+                now(),
+                true,
+                Vec::new(),
+                ON_SENT_SIZE,
+            ),
+            Instant::now(),
```

Comment:
Is there a way for `on_packet_sent` to take `now` out of the passed `SentPacket`, rather than passing it twice?

Reply:
We could, but would that be intuitive? The `SentPacket` might be sent before `now`. Given that it is an internal API and given that `now` is only an additional 4th argument, I would prefer not extracting current time from a related datastructure.

Do you feel strongly about it @larseggert?

Reply:
Don't feel strongly. Was just thinking about how to minimize the diff.